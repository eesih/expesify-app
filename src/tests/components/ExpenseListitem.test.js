import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListitem from '../../components/ExpenseListitem';
import expenses from '../fixtures/expenses';

test('should render ExpenseListitem', () => {
    const wrapper = shallow(<ExpenseListitem key={expenses[1].id} {...expenses[1]}/>);
    expect(wrapper).toMatchSnapshot();
});