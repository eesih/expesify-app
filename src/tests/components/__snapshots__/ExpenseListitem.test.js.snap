// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListitem 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    Rent
  </Link>
  <p>
    109500
  </p>
  <p>
    -345600000
  </p>
</div>
`;
